<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>pqHelpWindow</class>
 <widget class="QMainWindow" name="pqHelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Online Documentation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="contentsDock">
   <property name="windowTitle">
    <string>Contents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="searchDock">
   <property name="windowTitle">
    <string>Search</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionBackward"/>
   <addaction name="actionForward"/>
   <addaction name="actionSaveAsHomepage"/>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqHome.svg</normaloff>:/pqWidgets/Icons/pqHome.svg</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset resource="../pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqNavBack.svg</normaloff>:/pqWidgets/Icons/pqNavBack.svg</iconset>
   </property>
   <property name="text">
    <string>Go backward</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqNavForward.svg</normaloff>:/pqWidgets/Icons/pqNavForward.svg</iconset>
   </property>
   <property name="text">
    <string>Go forward</string>
   </property>
  </action>
  <action name="actionSaveAsHomepage">
   <property name="icon">
    <iconset resource="../pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqHomeWithSave.svg</normaloff>:/pqWidgets/Icons/pqHomeWithSave.svg</iconset>
   </property>
   <property name="text">
    <string>Save as homepage</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
